From bcad3e8b9878ec840da6abd182930bde9337980a Mon Sep 17 00:00:00 2001
From: Thomas Vandahl <tv@apache.org>
Date: Sun, 11 Nov 2012 16:04:19 +0000
Subject: [PATCH] Fix JCS-91: JCS fails to properly dispose of the lateral TCP
 listener thread. If a webapp that uses JCS lateral TCP cache is undeployed,
 the listener is left hanging with no hope for exit.

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/jcs/trunk@1408037 13f79535-47bb-0310-9956-ffa450edef68
---
 src/changes/changes.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/changes/changes.xml b/src/changes/changes.xml
index 7cfd8b68c..512827857 100644
--- a/src/changes/changes.xml
+++ b/src/changes/changes.xml
@@ -20,7 +20,7 @@
 	</properties>
 	<body>
 		<release version="2.0" date="unreleased" description="JDK 1.5 based major release">
-            <action dev="tv" type="fix" issue="JCS-94" due-to="Diego Rivera">
+            <action dev="tv" type="fix" issue="JCS-91" due-to="Diego Rivera">
                 JCS fails to properly dispose of the lateral TCP listener thread. 
                 If a webapp that uses JCS lateral TCP cache is undeployed, the listener is left 
                 hanging with no hope for exit.
